src/pages/admin/config-account/index.mdx

```diff
@@ -145,8 +145,8 @@ Seventh, create an access group to enable developers to access the <Globals name
 
 - Use the dev policies script
 
-    - Create a new [access group](https://cloud.ibm.com/docs/iam?topic=iam-account_setup), name it something like `<resource_group>-DEV` (all capital letters)
-    - Run the script `./terraform/scripts/acp-dev`, which will add the necessary policies to the access group
+    - Create a new [access group](https://cloud.ibm.com/docs/iam?topic=iam-account_setup), name it something like `<resource_group>-USERS` (all capital letters)
+    - Run the script `./terraform/scripts/acp-users`, which will add the necessary policies to the access group
     - Add the developers to the group
```

Comment:
Need to change this to something like `Add the users who will consume the cloud services to the group (e.g. developers, data scientists)`